admin_manual/configuration_files/external_storage/auth_mechanisms.rst

```diff
@@ -34,7 +34,8 @@ drawbacks are that sharing is disabled when this mechanism is in use, as
 ownCloud has no access to the storage credentials, and background file scanning 
 does not work.
 
-.. Note:: There is a workaround that allows sharing when using **Log-in credentials, save in session**, and that is using Ajax cron mode. (See :doc:`../../configuration_server/background_jobs_configuration`.)
+.. Note:: There is a workaround that allows sharing when using **Log-in credentials,
```

Comment:
should this also say "scan/detect" here instead of "sharing" ?
Because running ajax cron won't help with sharing in any way.